src/components/SpellingVariantInput.vue

```diff
@@ -84,6 +85,7 @@ const error = computed( () => {
 		),
 	};
 } );
+const helpUrl = 'https://www.wikidata.org/wiki/Help:Monolingual_text_languages'; // FIXME: inject?
```

Comment:
This should probably at least be [https://www.wikidata.org/wiki/Special:MyLanguage/Help:Monolingual_text_languages](https://www.wikidata.org/wiki/Special:MyLanguage/Help:Monolingual_text_languages), right? Not sure if it is worth injecting this link. It doesn't look like something that some 3rd party would want to point to someplace else, but not sure.

Thoughts?

Reply:
Agree about Special:MyLanguage; the process to get a language code added would be different on other wikis, though…

Reply:
Perhaps we can ask about it in the daily tomorrow?

Reply:
I don't think that there will be a daily tomorrow, but I'll bring it up with product

Reply:
Product suggestion is to put it into a message instead :heavy_check_mark: